lib/core/src/clientLogin.cpp

```diff
@@ -131,29 +123,6 @@ void setSessionSignatureClientside( char* _sig ) {
 
 } // setSessionSignatureClientside
 
-int printError( rcComm_t *Conn, int status, char *routineName ) {
-    rError_t *Err;
-    rErrMsg_t *ErrMsg;
-    int i, len;
-    if ( Conn ) {
-        if ( Conn->rError ) {
-            Err = Conn->rError;
-            len = Err->len;
-            for ( i = 0; i < len; i++ ) {
-                ErrMsg = Err->errMsg[i];
-                fprintf( stderr, "Level %d: %s\n", i, ErrMsg->msg );
-            }
-        }
-    }
-    char *mySubName = NULL;
-    const char *myName = rodsErrorName( status, &mySubName );
-    fprintf( stderr, "%s failed with error %d %s %s\n", routineName,
-             status, myName, mySubName );
-    free( mySubName );
-
-    return 0;
-}
-
```

Comment:
Let's leave the implementation of this function here.

People writing C/C++ clients may be using it and moving it from this location could break their code.

Reply:
can it be a wrapper/reference to just call the new/generic one?

Reply:
I understand the wrapper/reference idea, but I'm not aware of a new/generic implementation of this function.

Please elaborate.

Reply:
@korydraughn at the moment, `printError` is defined in in rcMisc.h but implemented in clientLogin.cpp. That doesn't feel right to me but I see your point.

Reply:
Oh. Thanks for the clarification.

If that's the case, then what you've done is correct.

Reply:
@Nishant2022 I think this can be marked as resolved.